tests/fixtures/scenarios/atmos-include-yaml-function/stacks/deploy/nonprod.yaml

```
@@ -0,0 +1,15 @@
+# yaml-language-server: $schema=https://atmos.tools/schemas/atmos/atmos-manifest/1.0/atmos-manifest.json
+
+vars:
+  stage: nonprod
+
+components:
+  terraform:
+    component-1:
+      metadata:
+        component: yaml-functions
+      vars:
+        string_var: !include stacks/mixins/vars.json .string_var
+        boolean_var: !include stacks/mixins/vars.yaml .boolean_var
+        list_var: !include stacks/mixins/vars.tfvars .list_var
+        map_var: !include stacks/mixins/vars.tfvars .map_var
```

The original intention of this funciton was to be able to load a varfile for an environment.

```suggestion
        map_var: !include stacks/mixins/vars.tfvars .map_var

    component-2:
      metadata:
        component: yaml-functions
      vars: !include stacks/mixins/vars.tfvars       
```